schemas/descriptors/label/descriptorLabel.schema.json

```diff
@@ -23,6 +23,21 @@
           "const": "xdm:descriptorLabel",
           "meta:xdmType": "string"
         },
+        "xdm:excludeProperty": {
+          "title": "Exclude Property",
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          ],
+          "description": "When present, the property of the source schema to which this descriptor excludes. This value is a JSON Pointer, applied to an instance of an object described by `xdm:sourceSchema`."
+        },
```

Comment:
@veramach-adobe , a quick question. Why this can't be just an array similar to xdm:sourceProperty which can take 1 or more elements. Also, if we are talking about the exclusion of nested properties, there is no way to enforce that in the JSON schema definition here that "excludeProperty" should have the parent mentioned in the "sourceProperty". It will be the responsibility of the consuming service to validate that. Right ?